style.css

```diff
@@ -100,3 +100,315 @@ header {
   text-decoration: none;
   list-style: none;
 }
+
+/*
+Works
+*/
+
+.works {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 5px;
+  background-color: white;
+  width: 85%;
+  height: auto;
+  margin: 114px auto;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-works {
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  flex-direction: column;
+  margin: 20px auto;
+}
+
+.snapshoot-img {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+}
+
+.leftblock {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 5px;
+  margin: 5px auto;
+  width: 295px;
+  overflow: hidden;
+}
+
+.ref {
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
- [ ] It's awesome 👍 that you have implemented a `button` tag for `See Project`. Kindly use hover and click event as per the project requirement.